### PR TITLE
Edge device files script & small README updates

### PR DESCRIPTION
**Pull request by t-fine**
edgeDeviceFiles:
- added the `-k` flag to create an API Key even if one is found with `$USER-Api-Key` name 
- changed env var from ICP_URL to CLUSTER_URL 
- added `-f` flag to move files to specified directory. I create it if it does not exist. 
- updated to use 4.0.0 tar file

Samples:
- some small readme updated based on some things I talked about with Parisa when we were walking thru what some uses will do next round of testing

### CODE REVIEW

File: tools/edgeDeviceFiles.sh [hunk 1]
```diff
@@ -142,12 +154,28 @@ function getClusterName () {
...
+
+# Create a IBM Cloud Pak platform API key
 function createAPIKey () {
-	echo "Creating IBM Cloud Private platform API key..."
-	echo "cloudctl iam api-key-create \"$EDGE_DEVICE API Key\" -d \"$EDGE_DEVICE API Key\" -f key.txt"
+	echo "Creating IBM Cloud Pak platform API key..."
+	echo "cloudctl iam api-key-create \"$USER-API-Key\" -d \"$USER API Key\" -f key.txt"
 
-	cloudctl iam api-key-create "$EDGE_DEVICE API Key" -d "$EDGE_DEVICE API Key" -f key.txt
+	cloudctl iam api-key-create "$USER-API-Key" -d "$USER API Key" -f key.txt
```
Comment (bmpotter) [inline tools/edgeDeviceFiles.sh]: I suggest the key name be a little more unique, e.g. `$USER-Edge-Device-API-Key`

Comment (t-fine) [reply tools/edgeDeviceFiles.sh]: Changed

File: tools/edgeDeviceFiles.sh [hunk 2]
```diff
@@ -197,46 +223,46 @@ function getICPCert () {
 
 # Locate the IBM Edge Computing for Devices installation content
 function gatherHorizonFiles () {
-	echo "Locating the IBM Edge Computing for Devices installation content for $EDGE_DEVICE device..."
-	echo "tar --strip-components n -zxvf ibm-edge-computing-x86_64-3.2.1.1.tar.gz ibm-edge-computing-x86_64-3.2.1.1/horizon-edge-packages/..."
+	echo "Locating the IBM Edge Computing Manager for Devices installation content for $EDGE_DEVICE device..."
+	echo "tar --strip-components n -zxvf ibm-edge-computing-x86_64-4.0.0.tar.gz ibm-edge-computing-x86_64-4.0.0/horizon-edge-packages/..."
```
Comment (bmpotter) [inline tools/edgeDeviceFiles.sh]: In the future, we probably want the version to be something that can be input via flag or env var, so the script can work for fix packs, etc.

Comment (t-fine) [reply tools/edgeDeviceFiles.sh]: Ah yes, gotcha. I'll make a note of this

**Pull request merged**